omnibus_build.sh

```diff
@@ -9,11 +9,16 @@
 PROJECT_DIR=dd-agent-omnibus
 PROJECT_NAME=datadog-agent
 LOG_LEVEL=${LOG_LEVEL:-"info"}
+JMX_VERSION=${JMX_BRANCH:-"current"}
```

Comment:
as discussed let's remove this altogether: the jmxfetch version should be the one that's defined in the agent's `config.py` on the branch that we're building.